### PR TITLE
Added search terms for Arts and Crafts shops

### PR DESCRIPTION
**Pull request by ProgramminCat**

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -16694,3 +16694,8 @@ uk:Центр мистецтв
 vi:Trung tâm nghệ thuật
 zh-Hans:艺术中心
 zh-Hant:藝術中心
+
+shop-craft
```
Comment (pastk) [inline data/categories.txt:16698]:
```suggestion
shop-craft|@shop
```
It provides "shop"/"store" etc. synonyms automatically, so all lang-specific lines shouldn't contain them (pls check how synonyms are done for other shops).

Please take existing `[type.shop.craft]` translations from `types_strings.txt` as a starting point for all languages.

**Review by pastk — CHANGES_REQUESTED** [review]

File: data/categories.txt [hunk 2]
```diff
@@ -16694,3 +16694,8 @@ uk:Центр мистецтв
 vi:Trung tâm nghệ thuật
 zh-Hans:艺术中心
 zh-Hant:藝術中心
+
+shop-craft|@shop
+en:Arts and Crafts Shop|Craft Shop|Crafts Shop
```
Comment (vng) [inline data/categories.txt:16699]: "en:Craft" is enough here. And other translations are the same.

File: data/categories.txt (hunk 1 above)
Comment (pastk) [reply data/categories.txt]: Please take existing [type.shop.craft] translations from types_strings.txt as a starting point for all languages.

Comment (pastk) [reply data/categories.txt]: @ProgramminCat just checking to make sure - have you seen this comment?

Comment (ProgramminCat) [reply data/categories.txt]: > @ProgramminCat just checking to make sure - have you seen this comment?

Yes I have. I will try that soon

**Comment by vng** [comment]
Please, rebase and squash to a single commit.

**Comment by ProgramminCat** [comment]
> Please, rebase and squash to a single commit.

I'm not familiar with those features, could you please tell me how to "rebase" and "squash"?

**Comment by vng** [comment]
Ok, I did it. It is not as simple if you are not a developer :(

**Review by vng — APPROVED** [review]

**Comment by vng** [comment]
@pastk I don't want to take shop.craft from type_strings, because of again _wide terms range_ :)

**Comment by pastk** [comment]
> @pastk I don't want to take shop.craft from type_strings, because of again _wide terms range_ :)

I understand, but again its important to be able to search using a name from type_strings (because this is what users see when they select a POI). Otherwise it becomes a guess game for users.
Is it possible to maybe pre-process search queries for full type name matches first? And only then process partial token matches, etc.

**Comment by pastk** [comment]
@organicmaps/translations please add synonyms for Arts & Crafts Supplies/Store POI for your language. They should be short and without "store/shop" words (as they will be included automatically) - this is the reason we can't take existing POI type translations directly.

**Comment by muralito** [comment]
Please explain if  in the same "OM term" you want to refer to a shop that sell crafts, or where artisans sells the objects they produce, and also to the shop that sell supplies to make arts and crafts?

In OSM shop=craft refers only to the second. See https://wiki.openstreetmap.org/wiki/Tag:shop%3Dcraft
"Artesania" in spanish, at least locally, refers to the 1st.

**Comment by pastk** [comment]
> he shop that sell supplies to make arts and crafts?
> 
> In OSM shop=craft refers only to the second. See https://wiki.openstreetmap.org/wiki/Tag:shop%3Dcraft

Only that.

(shops selling arts & crafts are tagged as [shop=art](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dart) as they have search synonyms already)

File: data/categories.txt [hunk 3]
```diff
@@ -16737,3 +16737,10 @@ tr:hapishane
 uk:в'язниця
 vi:nhà tù
 zh-Hans:监狱
+
+shop-craft|@shop
```
Comment (pastk) [inline data/categories.txt:16741]:
```suggestion
# A shop that sell supplies to make arts and crafts (different from shop-arts that sells works of art)
shop-craft|@shop
```

File: data/categories.txt [hunk 4]
```diff
@@ -16737,3 +16737,10 @@ tr:hapishane
...
+en:Craft
+be:Рамяство
+fr:Artisanat
+es:Artesanía
+ru:Ремесло
```
Comment (JohannesDeml) [inline data/categories.txt:16747]:
```suggestion
ru:Ремесло
de:Künstlerbedarf
```

File: data/categories.txt [hunk 5]
```diff
@@ -16737,3 +16737,12 @@ tr:hapishane
...
 vi:nhà tù
 zh-Hans:监狱
+
+# A shop that sell supplies to make arts and crafts (different from shop-arts that sells works of art)
+shop-craft|@shop
+en:Craft
+be:Рамяство
+fr:Artisanat
+es:Artesanía
+ru:Ремесло
```
Comment (pastk) [inline data/categories.txt:16747]:
```suggestion
ru:Ремесло|товары для творчества
```
@vng would that work?

**Comment by patepelo** [comment]
> Please explain if  in the same "OM term" you want to refer to a shop that sell crafts, or where artisans sells the objects they produce, and also to the shop that sell supplies to make arts and crafts?
> 
> In OSM shop=craft refers only to the second. See https://wiki.openstreetmap.org/wiki/Tag:shop%3Dcraft
> "Artesania" in spanish, at least locally, refers to the 1st.

"Tienda de manualidades" is more accurate

File: data/categories.txt [hunk 6]
```diff
@@ -16737,3 +16737,12 @@ tr:hapishane
 uk:в'язниця
 vi:nhà tù
 zh-Hans:监狱
+
+# A shop that sell supplies to make arts and crafts (different from shop-arts that sells works of art)
+shop-craft|@shop
```
Comment (andergd) [inline data/categories.txt:16742]: `eu:eskulanak`

**Comment by matheusgomesms** [comment]
> @organicmaps/translations please add synonyms for Arts & Crafts Supplies/Store POI for your language. They should be short and without "store/shop" words (as they will be included automatically) - this is the reason we can't take existing POI type translations directly.

I don't understand, sorry. We have to provide a translation for Arts & Crafts Store without the word "store" in it, right? And it has to be short too?

**Comment by vng** [comment]
@matheusgomesms A phrase on how you would search OSM shop=craft store on a map in PT. In English it is a "craft shop", but the "shop" world (and "store") is already inherited from the ```@shop``` category. Can keep it or can skip it.

**Comment by pastk** [comment]
> 
> I don't understand, sorry. We have to provide a translation for _Arts & Crafts Store_ without the word "store" in it, right? And it has to be short too?

Actually its more like a store with Supplies/Goods for Making Arts & Crafts, it should be distinct from a store that sells ready-made works of art/craft.

Re why a "store/shop" keyword should be omitted for all shops - see #5323.
(TBH IMHO its better to bear with duplicated "shop" keywords everywhere rather than spend energy to explain why it should be omitted in every other PR :))

**Comment by matheusgomesms** [comment]
Thanks! For PT then:

`pt:3Manualidades|5Armarinho`

**Pull request merged**